substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -300,25 +300,46 @@ impl<T: Config> Pallet<T> {
 	fn make_payout(
 		stash: &T::AccountId,
 		amount: BalanceOf<T>,
-	) -> Option<(PositiveImbalanceOf<T>, RewardDestination<T::AccountId>)> {
-		let maybe_imbalance = match Self::payee(stash) {
-			RewardDestination::Controller => Self::bonded(stash)
-				.map(|controller| T::Currency::deposit_creating(&controller, amount)),
-			RewardDestination::Stash => T::Currency::deposit_into_existing(stash, amount).ok(),
-			RewardDestination::Staked => Self::bonded(stash)
-				.and_then(|c| Self::ledger(&c).map(|l| (c, l)))
-				.and_then(|(controller, mut l)| {
-					l.active += amount;
-					l.total += amount;
-					let r = T::Currency::deposit_into_existing(stash, amount).ok();
+	) -> Option<(PositiveImbalanceOf<T>, PayoutDestination<T::AccountId>)> {
+		// NOTE: temporary getter while `Payee` -> `Payees` lazy migration is taking place.
```

Comment:
nit: I'd probably add the tracking issue link for the lazy migration in the comments.